src/goto-programs/goto_program_template.h

```diff
@@ -80,60 +79,129 @@ class goto_program_templatet
     return *this;
   }
 
-  /*! \brief Container for an instruction of the goto-program
-  */
+  /// This class represents a node in the control flow graph of the function.
```

Comment:
This is easily misleading: we have a separate class for control-flow graphs. Goto programs are an intermediate representation where control-flow is very simple to deduce - but they aren't control-flow graphs.